pages/checkout.vue

```diff
@@ -521,17 +517,18 @@ export default {
         case stepLabels.GO_TO_SHIPPING: {
           if (!isValidPersonalDetails.value) return
           await savePersonalDetails()
+          enableNextStep.value = false
```

Comment:
why are we setting to false here?

Reply:
When we are redirecting to the next step from the valid step. The checkout button should be disabled until that step is not valid so it's added here.